src/styles/index.less

```diff
@@ -5,6 +5,7 @@
 @import '../../node_modules/cnvs/styles/cnvs.less'; // Source CNVS
 @import 'vendor/cnvs/styles.less'; // CNVS Overrides
 @import 'vendor/loaders.less';
+@import '../../plugins/design-system/styles/styles.less'; // TODO: Tommy figure out where to include this.
```

Comment:
What's the problem with that one?

Reply:
We want to be able to use CNVS styles for plugins. The point was discussed and it was decided this way is fine. I will remove the comment.